test/processors/concerns/http_files_test.rb

```diff
@@ -37,7 +37,7 @@ def logger
 
   it 'can download a small file' do
     stub_request(:get, small_uri.to_s).
-      with(headers: { 'Host' => 'test.prx.org:80' } ).
+      with(headers: { 'Host' => 'test.prx.org' } ).
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.